lib/main_publiccloud.pm

```diff
@@ -221,6 +221,10 @@ sub load_publiccloud_tests {
             load_publiccloud_download_repos();
         } elsif (get_var('PUBLIC_CLOUD_QAM')) {
             load_maintenance_publiccloud_tests();
+        } elsif (get_var('PUBLIC_CLOUD_AISTACK')) {
+            my $args = OpenQA::Test::RunArgs->new();
+            loadtest('publiccloud/upload_image', run_args => $args);
+            loadtest('publiccloud/aistack', run_args => $args);
```

Comment:
You forgot to include this module.

Reply:
Indeed, missing

Reply:
Im confused. Im loading test for publiccloud/aistack. Which is another PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20015/files

Reply:
ah then you have before to merge the pr 20015, for that module: this 20384 needs for it.